clang/utils/perf-training/llvm-support/build.test

```diff
@@ -0,0 +1,2 @@
+RUN: %cmake -G %cmake_generator -B %t -S %llvm_src_dir -DCMAKE_C_COMPILER=%clang -DCMAKE_CXX_COMPILER=%clang -DCMAKE_CXX_FLAGS="--driver-mode=g++" -DCMAKE_BUILD_TYPE=Release
```

Comment:
Why the `--driver-mode=g++` flag here?

Reply:
The current bolt.lit.cfg substitutes `%clang_cpp` to `clang --driver-mode=g++` because there is no clang++ symlink to the instrumented bolt binary.  So this option is required in order to use `clang` as a c++ compiler.  The other alternative solution would be to add the clang++ symlink.

Reply:
Ah, makes sense. Seems reasonable enough to me!